third_party/go/BUILD

```diff
@@ -8,7 +8,6 @@ go_toolchain(
         "01cd67bbc12e659ff236ecebde1806f76452f7ca145c172d5ecdbf4f4803daae",  # freebsd_amd64
         "e85278e98f57cdb150fe8409e6e5df5343ecb13cebf03a5d5ff12bd55a80264f",  # linux_amd64
     ],
-    strip_srcs = CONFIG.BUILD_CONFIG != "dbg",
```

Comment:
No, it will never strip sources as the default is false. We need this now because we use this toolchain to bootstrap `please_go`, and `go build` doesn't work without the sources.